Source/WarpX.H

```diff
@@ -616,6 +620,39 @@ public:
     void MacroscopicEvolveE (int lev, amrex::Real dt);
     void MacroscopicEvolveE (int lev, PatchType patch_type, amrex::Real dt);
 
+    /**
+     * \brief
+     * Function to update the E-field using Ohm's law (hybrid-PIC model).
+     */
+    void HybridPICSolveE (                               DtType dt_type);
+    void HybridPICSolveE (const int lev,                       DtType dt_type);
+    void HybridPICSolveE (const int lev, PatchType patch_type, DtType dt_type);
```

Comment:
Could these functions become members of the `HybridModel` class?
This could provide more encapsulation and make it clearer which members go along with with solver/scheme, as well as de-clutter the `WarpX.H` file.
Note that, as a consequence, you may need to pass the `Efield_fp`, `Bfield_fp` as argument to these function.